app/assets/stylesheets/set_color.scss

```diff
@@ -1,3 +1,4 @@
 $gray-medium-light: #eaeaea;
-$red: red;
+$red: #ff0000;
```

Comment:
can you write color code briefer?

Reply:
I write it because I want this file was be defined same type :))))